esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -345,6 +345,28 @@ define([
 
             //  Implementation  ---
             getFilter: function () {
+                var context = this;
```

Comment:
No need for "context" here.  It is only needed when you need to refer to "this" inside a lambda function...

Reply:
I refer to "this" in the switch case when sorting occurs in this function, that is why i am using <var context = this;>

Reply:
Do we not use "context" anymore to refer out of scope calls?